Sources/StreamChatUI/Utils/UITextView+Extensions.swift

```diff
@@ -5,31 +5,17 @@
 import UIKit
 
 extension UITextView {
-    ///
-    /// Reference:
-    /// https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/TextLayout/Tasks/StringHeight.html
-    ///
-    func calculatedTextHeight() -> CGFloat {
-        // Height is not calculated correctly with empty text
-        let string: String = text.isEmpty ? " " : text
-        let textStorage = NSTextStorage(string: string)
-        let width = frame.width - textContainerInset.right - textContainerInset.left
-        let customTextContainer = NSTextContainer(size: .init(width: width, height: CGFloat.greatestFiniteMagnitude))
-        let layoutManager = NSLayoutManager()
-        
-        layoutManager.addTextContainer(customTextContainer)
-        
-        textStorage.addLayoutManager(layoutManager)
-        if let font = font {
-            textStorage.addAttribute(.font, value: font, range: .init(0..<textStorage.length))
+    func highlightMention(mention: String) {
+        let attributeText = NSMutableAttributedString(attributedString: attributedText)
+        let string = attributeText.string
+        guard let range = string
+            .range(of: mention, options: .caseInsensitive)
+            .map({ NSRange($0, in: string) })
+        else {
```

Comment:
weird formatting again

Reply:
It is normal to pipe functions at the bottom in a new line in Swift. Just like we do in the E2E Tests. 

But same as above, if we want to be more strict here, we should introduce rules in SwiftFormat